.vscode/settings.json

```diff
@@ -25,11 +25,8 @@
 		"editor.rulers": [150]
 	},
 	// Lets you preview .dmi files in the editor
-	"workbench.editorAssociations": [
```

Comment:
what's up with this unrelated file change?

Reply:
I have no idea but vscode is forcing it into any branch I try and edit.

Reply:
I'll fix this.